modules/oe_content_consultation/config/install/field.field.node.oe_consultation.oe_consultation_documents.yml

```diff
@@ -0,0 +1,35 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_consultation_documents
+    - node.type.oe_consultation
+    - oe_content_entity_document_reference.oe_document_reference_type.oe_document
+    - oe_content_entity_document_reference.oe_document_reference_type.oe_publication
+  module:
+    - composite_reference
+    - entity_reference_revisions
+third_party_settings:
+  composite_reference:
+    composite: true
+id: node.oe_consultation.oe_consultation_documents
+field_name: oe_consultation_documents
+entity_type: node
+bundle: oe_consultation
+label: Documents
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:oe_document_reference'
+  handler_settings:
+    target_bundles:
+      oe_document: oe_document
+      oe_publication: oe_publication
+    sort:
+      field: _none
+    auto_create: false
+    auto_create_bundle: oe_document
```

Comment:
Don't need.

Reply:
Yes. I know that this line in the config should not have an impact, but it was good to remove it.